AmazonChimeSDK/AmazonChimeSDK/audiovideo/video/DefaultVideoTile.swift

```diff
@@ -20,13 +20,14 @@ import VideoToolbox
                 attendeeId: String,
                 videoStreamContentWidth: Int,
                 videoStreamContentHeight: Int,
+                pauseState: VideoPauseState,
```

Comment:
Probably want to avoid the backwards incompatible change and just use `setPauseState` for now.

Reply:
Another way to avoid breaking change is to overload the initializer and have both `init(tileId, attendeeId, width, height, isLocal, logger)` and `init(tileId, attendeeId, width, height, pauseState, isLocal, logger)`.

Reply:
That works too (on android we use optional params quite a bit too).  I do wonder if maybe in the future we should break out the builder pattern, or just setting the values on the struct directly :p .

Reply:
The old initializer should call the new initializer with `pauseState = .unpaused`.

Reply:
Also I feel like probably Android and iOS should follow the same pattern to resolve this issue, either both overload the initializer (or option params), or both call `setPauseState()`.